### DISCUSSION TITLE
RA8 Vision Board Questions Collection

### DISCUSSION THREAD

**Discussion by Cathy-lulu** [RT-Thread/rt-thread#8812] [General]
For engineers who have purchased the RA8 Vision Board or are interested in OpenMV and RT-Thread RTOS, this is the channel for us to discuss and brainstorm new ideas.

**Comment by jarain78** [comment]
Hello, I have a question. I'm programming my Vision Board with OpenMV, and I want to use a .tflite model from the SD memory. However, when I run the program, OpenMV returns the following error: Exception: Failed to load "trained.tflite", did you copy the .tflite and labels.txt file onto the mass-storage device? (1)

Does anyone know if it's necessary to update the firmware, and if so, which version needs updating?

Regards,

Jaime

**Reply by Rbb666** [reply]
1、You need to make sure the sd card is fat32, and you can check the shell terminal to make sure that the sd card can be mounted and the sd card file system can be accessed
2、If it still doesn't work, you can try updating the openmv firmware：https://github.com/RT-Thread-Studio/sdk-bsp-ra8d1-vision-board/tree/master/projects/vision_board_openmv/firmware

**Reply by jarain78** [reply]
Thanks for your help, I have managed to fix the problem with the SD card. However, I have another question, do you happen to have a high resolution image showing the pins on the vision card? The image I found on the internet is distorted when I zoom in trying to see the pin numbers.

**Reply by Rbb666** [reply]
Maybe you can find the corresponding pins more clearly in this picture:https://github.com/RT-Thread-Studio/sdk-bsp-ra8d1-vision-board/blob/master/documents/images/postcard.png
